kitsune/search/api.py

```diff
@@ -54,7 +54,8 @@ def _question_suggestions(searcher, text, locale, product, max_results):
         model='questions_question',
         question_is_archived=False,
         question_is_locked=False,
-        question_has_helpful=True)
+        question_has_helpful=True,
```

Comment:
should we remove this has_helpful filter? I'm not sure if it's possible that an a question gets solved but doesn't have any answers marked helpful.